superdesk/storage/fix_links.py

```diff
@@ -56,8 +57,11 @@ def run(self, prefix, resource, dry_run):
                     if dry_run:
                         print("update", item["_id"], updates.keys())
                     else:
-                        print(".", end="")
-                        service.system_update(item["_id"], updates, orig)
+                        try:
+                            service.system_update(item["_id"], updates, orig)
+                            print(".", end="")
+                        except SuperdeskApiError:
+                            print("x", end="")
```

Comment:
It might be good for debugging purposes if the error were to be logged to the console for further investigation (with resource name/id and reason for error included in the output)